README.md

```diff
@@ -197,9 +197,9 @@ version: '3.8'
 
 services:
   mailserver:
-    image: docker.io/mailserver/docker-mailserver:latest
```

Comment:
I would like to keep this as this makes clear that only with DockerHub, we use `mailserver/docker-mailserver`. Another possible address would be `ghcr.io/docker-mailserver/docker-mailserver`.

Reply:
> Another possible address would be ghcr.io/docker-mailserver/docker-mailserver.

For now, it needs auth..
It's a big blocker

Will update all examples to docker.io then ;)

Reply:
the ghcr needs auth for pulling? Never noticed 🙃

Reply:
@williamdes yes, we should stick to `docker.io` :) But GHCR does not need auth AFAIK as @wernerfred noticed too :D